tests/unit_tests/test_game.py

```diff
@@ -405,6 +406,8 @@ def custom_game(event_loop, database, game_service, game_stats_service):
             assert new_rating != Rating(*player.ratings[RatingType.GLOBAL])
 
 
+# FIXME - discuss correct get_army_result behaviour
```

Comment:
So this test now fails because the first result (with one reported victory and one reported defeat) now is reported as an unknown outcome? It might be a good idea to unrank the game due to `UNKNOWN_RESULT` in this case as well.

Reply:
I'll change the test to do that.